modules/exploits/linux/http/zencart_authenticated_rce.rb

```diff
@@ -0,0 +1,205 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+###
+#
+#
+# This exploit  write payload in database and trig to command
+# a bug in an zencart v1.7.5b web application
+#
+###
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
```

Comment:
Rather than call `check` inside `exploit` you can use the `AutoCheck` mixin. The `AutoCheck` mixin will call `check` automatically.

```suggestion
  include Msf::Exploit::Remote::HttpServer
  prepend Msf::Exploit::Remote::AutoCheck
```